evm/src/cpu/kernel/tests/fields.rs

```diff
@@ -0,0 +1,200 @@
+use anyhow::Result;
+use ethereum_types::U256;
+use rand::{thread_rng, Rng};
+
+use crate::cpu::kernel::aggregator::combined_kernel;
+use crate::cpu::kernel::interpreter::run_with_kernel;
+
+const P254: u32 = 107;
```

Comment:
Please add a comment to explain why our prime is `107` and when it should be changed and by whom 😛